doc/weakly_supervised.rst

```diff
@@ -151,18 +151,50 @@ tuples you're working with (pairs, triplets...). See the docstring of the
 Algorithms
 ==================
 
+
 ITML
 ----
 
-Information Theoretic Metric Learning, Davis et al., ICML 2007
+Information Theoretic Metric Learning(:py:class:`ITML <metric_learn.itml.ITML>`)
 
-`ITML` minimizes the differential relative entropy between two multivariate
-Gaussians under constraints on the distance function, which can be formulated
-into a Bregman optimization problem by minimizing the LogDet divergence subject
-to linear constraints. This algorithm can handle a wide variety of constraints
+`ITML` minimizes the (differential) relative entropy, aka Kullback–Leibler 
+divergence, between two multivariate Gaussians subject to constraints on the 
+associated Mahalanobis distance, which can be formulated into a Bregman 
+optimization problem by minimizing the LogDet divergence subject to 
+linear constraints. This algorithm can handle a wide variety of constraints
 and can optionally incorporate a prior on the distance function. Unlike some
-other methods, ITML does not rely on an eigenvalue computation or semi-definite
-programming.
+other methods, `ITML` does not rely on an eigenvalue computation or 
+semi-definite programming.
+
+
+Given a Mahalanobis distance parameterized by :math:`A`, its corresponding 
+multivariate Gaussian is denoted as:
+
+.. math::
+    p(\mathbf{x}; \mathbf{A}) = \frac{1}{Z}\exp(-\frac{1}{2}d_\mathbf{A}
+    (\mathbf{x}, \mu)) 
+    =  \frac{1}{Z}\exp(-\frac{1}{2}((\mathbf{x} - \mu)^T\mathbf{A}
+    (\mathbf{x} - \mu)) 
+
+where :math:`Z` is the normalization constant, the inverse of Mahalanobis 
+matrix :math:`\mathbf{A}^{-1}` is the covariance of the Gaussian.
+
+Given pairs of similar points :math:`S` and pairs of dissimilar points 
+:math:`D`, the distance metric learning problem is:
+
+.. math::
+
+    \min_\mathbf{A} \textbf{KL}(p(\mathbf{x}; \mathbf{A}_0) || p(\mathbf{x}; 
```

Comment:
it would be better to write the optimization problem in terms of the logdet divergence, which gives a more "concrete" formulation than the KL version

Reply:
done